composer.json

```diff
@@ -13,11 +13,11 @@
         "http-interop/http-factory-guzzle": "^1.0",
         "knplabs/github-api": "^3.4",
         "m4tthumphrey/php-gitlab-api": "^11.5",
-        "nikic/php-parser": "4.13.*",
+        "nikic/php-parser": "4.14.*",
         "phpstan/phpstan": "^1.2",
         "spryker-sdk/integrator": "dev-develop",
         "spryker-sdk/sdk-contracts": "^0.2.0",
-        "spryker/code-sniffer": "^0.17.3",
+        "spryker/code-sniffer": "dev-master",
```

Comment:
Why are those in require and not require-dev?

Reply:
Hi @kalaputsik,
could you please check why it's necessary to have code sniffer in the required section?
Thank you!

Reply:
Hi @alexanderM91, @Kyrylo-Meretskyi ,
I suppose nowadays we shouldn't require it. 
Previously, when we used the Code Sniffers for the evaluation process it made sense, but now we might move it to the dev

Reply:
Fixed, spryker/code-sniffer moved to dev dependencies